dask/dataframe/tests/test_dataframe.py

```diff
@@ -4429,3 +4429,17 @@ def test_fuse_roots():
     res = ddf1.where(ddf2)
     hlg = fuse_roots(res.__dask_graph__(), keys=res.__dask_keys__())
     hlg.validate()
+
+
+def test_attrs():
+    df = pd.DataFrame({"A": [1, 2], "B": [3, 4], "C": [5, 6]})
+    df.attrs = {"date": "2020-10-16"}
+    df.A.attrs["unit"] = "kg"
```

Comment:
I wouldn't recommend setting the attrs on a Series like this. It's not clear to me that it's a case supported by pandas (indexing into a DataFrame and setting on a Series).

Reply:
I don't think it would be consistent with the rest of the DataFrame methods if it wasn't supported. 
Why should `df.A[0] = 10` or `df.A.values[0] = 100` or `df.A.name = "A_new"` be allowed but not `df.A.attrs["unit"] = "kg"`?

What way were you thinking? I'm not even sure how to do this in any other way to be honest. I've never initialized series separately and then appended them to a dataframe.

Reply:
There's some discussion at https://github.com/pandas-dev/pandas/issues/35425, but let's not focus on it here.

For this PR we just need two tests. One on a `dd.from_pandas(dataframe_with_attrs)` and a second test with `dd.from_pandas(series_with_attrs)`.